src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -252,6 +252,7 @@ internal enum ErrorCode
         ERR_GenericConstraintNotSatisfiedTyVar = 314, // Requires SymbolDistinguisher.
         ERR_GenericConstraintNotSatisfiedValType = 315, // Requires SymbolDistinguisher.
         ERR_DuplicateGeneratedName = 316,
+        ERR_You_must_provide_an_initializer_for_a_readonly_variable = 317,
```

Comment:
Please follow existing naming style. Also, including message wording in the name is probably a bad idea. The name should reflect the cause instead, i.e. a missing initializer. For example, ERR_ReadonlyVariableWithoutInitializer.